docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: '3.2'
+services:
```

Comment:
If we have three independent image components in the composite: KBS, AS and RVPS, it will be clearer. But at present, KBS lacks the module for grpc communication with AS. I can implement this in the following PR.

Reply:
Now the AS is embedded in KBS here https://github.com/confidential-containers/kbs/blob/main/src/api_server/src/lib.rs#L89 as a lib. Do you mean that we should implement a proxy inside KBS to connect to AS?